app/main.py

```diff
@@ -1,6 +1,28 @@
 from fastapi import FastAPI
 from fastapi.middleware.cors import CORSMiddleware
+import uvicorn
+from langchain_community.document_loaders import CSVLoader
+from langchain_community.vectorstores import Chroma
+from langchain_community.embeddings import HuggingFaceEmbeddings
+import traceback
+import csv
+from models.api_models import Prompt
+from models.izakaya import Izakaya
+from models.coordinate import Coordinate
```

Comment:
FastAPIのダメな仕様なんですが相対パスにするとデプロイ時にモジュールのインポートに失敗するという最悪すぎるエラーが起きるので絶対参照にしておいて欲しいです（FastAPIを許すな）
```suggestion
from app.models.api_models import Prompt
from app.models.izakaya import Izakaya
from app.models.coordinate import Coordinate
```

Reply:
了解！
ほかのレビューで指摘されてたけど、`uvicorn app.main:app --reload`で実行するようにしたら起動できた！
感謝！